src/compiler/bundle/ext-transforms-plugin.ts

```diff
@@ -164,7 +164,7 @@ export const extTransformsPlugin = (
         // Set style docs
         if (cmp) {
           cmp.styleDocs ||= [];
-          mergeIntoWith(cmp.styleDocs, cssTransformResults.styleDocs, (docs) => docs.name);
+          mergeIntoWith(cmp.styleDocs, cssTransformResults.styleDocs, (docs) => `${docs.name},${docs.mode}`);
```

Comment:
This is _probably_ OK as far as using a comma as the joiner here. I don't think any css property nor mode would ever have a comma in it, LMK if you think this isn't safe. Regardless, it would be easy to change in the future, as it's not exposed to users in our output

Reply:
yeah I think it should be fine because I think even if there were a comma somewhere then the string produced should still be unique (that little function is only used for identifying when items are the same item)